worldengine/World.proto

```diff
@@ -1,3 +1,4 @@
+syntax = "proto2";
```

Comment:
`World.proto` has to be changed a bit to make it work for "proto3", I tried. I also found the documentation. ;)
So should I rather use "proto3" or is that something to be left for the future? I assume that the save-files won't be compatible between proto2 and 3 (the result is different according to git).